packages/matter.js-tools/package.json

```diff
@@ -55,7 +55,8 @@
         "yargs": "^17.7.2"
     },
     "optionalDependencies": {
-        "v8-profiler-next": "^1.10.0"
+        "v8-profiler-next": "^1.10.0",
+        "@esbuild/darwin-x64": "^0.20.1"
```

Comment:
If this causes the optional dependencies to be locked when node_modules is present but package_lock.json is not then we should include all of esbuild's platform dependencies here although I'm not sure how important AIX support is. :)  Otherwise we should probably take this out and perhaps have npm install abort if package-lock.json is missing but node_modules is present.

Reply:
will be fixed by new PR